content/en/shokometadata/install.md

```diff
@@ -59,14 +62,106 @@ Move both of these folders into the following directory.
 
 Proceed to **Editing Plex Scanners** to make the required edits for the scanners to work. 
 
-##### Linux
+### Linux
+
+**Before you continue**, to make it easier to follow the guide, regardless of the Plex install method, we will assign an environment
+variable (`$PLEX_HOME`) that will point to the base directory for your Plex content.
+
+Below are the three most common install methods, and what you need to do for each one before you continue with the rest of the guide. If you
+installed Plex using another method, then please assign the variable to your own, and be sure to use the correct user when executing the
+steps listed below.
+
+##### Bare-metal install
+
+Sign in with **the user responsible for running PMS** (be sure to replace `$USERNAME` with the username or user-id),
+```sh
+sudo su --login --shell=/bin/bash $USERNAME
+```
+
+and assign thes variable.
+```sh
+export PLEX_HOME="/var/lib/plex/Plex Media Server"
```

Comment:
While this is usually the install directory, it isn't always the case for all installs, might be worth noting that

Reply:
Will add a note for this(/these) install method(s).